src/Components/AdvancedSearch/Internal/QueryBuilder/QueryBuilder.styles.ts

```diff
@@ -0,0 +1,182 @@
+import { StylesConfig } from 'react-select';
+import {
+    IQueryBuilderStyleProps,
+    IQueryBuilderStyles,
+    IQueryBuilderRowStyleProps,
+    IQueryBuilderRowStyles
+} from './QueryBuilder.types';
+
+/**
+ * Query builder container styles
+ */
+export const classPrefix = 'cb-querybuilder';
+const classNames = {
+    root: `${classPrefix}-root`,
+    headerGrid: `${classPrefix}-headerGrid`,
+    headerText: `${classPrefix}-headerText`,
+    rowContainer: `${classPrefix}-rowContainer`
+};
+
+export const getStyles = (
+    props: IQueryBuilderStyleProps
+): IQueryBuilderStyles => {
+    return {
+        root: [classNames.root],
+        headerGrid: [
+            classNames.headerGrid,
+            {
+                display: 'grid',
+                width: '100%',
+                columnGap: '20px'
+            },
+            props.rowCount === 1
+                ? {
+                      gridTemplateColumns: '4fr 4fr 4fr 1fr'
+                  }
+                : {
+                      gridTemplateColumns: '2fr 4fr 4fr 4fr 1fr'
+                  }
+        ],
+        headerText: [
```

Comment:
The header text should not be bright white. Let's make it match the designs please. It should be more of a softer gray

Reply:
header text in the designs is white